arm-recoveryservicesbackup/2016-06-01/swagger/recoveryservicesbackup.json

```diff
@@ -0,0 +1,1548 @@
+            "in": "path",
+            "description": "OperationID which represents the operation",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK"
+          },
+          "202": {
+            "description": "Accepted"
+          },
+          "204": {
+            "description": "NoContent"
+          }
+        },
+        "deprecated": false
+      }
+    }
+  },
+  "definitions": {
+    "Object": {
+      "description": "Base of all objects.",
+      "type": "object",
+      "properties": {}
+    },
+    "OperationStatus": {
+      "description": "Operation status.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "ID of the operation.",
+          "type": "string"
+        },
+        "name": {
+          "description": "Name of the operation.",
+          "type": "string"
+        },
+        "status": {
+          "description": "Operation status. Possible values: InProgress, Succeeded, Failed, Canceled.",
+          "type": "string"
+        },
+        "startTime": {
+          "format": "date-time",
+          "description": "Operation start time.",
+          "type": "string"
+        },
+        "endTime": {
+          "format": "date-time",
+          "description": "Operation end time.",
+          "type": "string"
+        },
+        "error": {
+          "$ref": "#/definitions/OperationStatusError",
+          "description": "Error information related to this operation."
+        },
+        "properties": {
+          "$ref": "#/definitions/OperationStatusExtendedInfo",
+          "description": "Additional information associated with this operation."
+        }
+      }
+    },
+    "OperationStatusError": {
+      "description": "Error information associated with operation status call.",
+      "type": "object",
+      "properties": {
+        "code": {
+          "description": "Error code of the operation failure.",
+          "type": "string"
+        },
+        "message": {
+          "description": "Error message displayed if the operation failure.",
+          "type": "string"
+        }
+      }
+    },
+    "OperationStatusExtendedInfo": {
+      "description": "Base class for additional information of operation status.",
+      "type": "object",
+      "properties": {}
+    },
+    "ResourceList[Job]": {
+      "description": "Resource list wrapper for \"T\" type",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Gets or sets the list of resources.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Resource[Job]"
+          }
+        },
+        "nextLink": {
+          "description": "Gets or sets the uri to fetch the next page of resources. Call ListNext() fetches next page of resources.",
+          "type": "string"
+        }
+      }
+    },
+    "Resource[Job]": {
```

Comment:
This should be named as "Job".
All the generic resource properties should be modeled in the "Resource" model definition with location being required as per the ARM spec. Take a look at the arm-storage spec from this repo as an example